rules/camunda-cloud/signal-reference.js

```diff
@@ -14,7 +14,7 @@ const { skipInNonExecutableProcess } = require('../utils/rule');
 
 module.exports = skipInNonExecutableProcess(function() {
   function check(node, reporter) {
-    if (!isAny(node, [ 'bpmn:StartEvent', 'bpmn:IntermediateThrowEvent', 'bpmn:EndEvent' ])) {
+    if (!isAny(node, [ 'bpmn:StartEvent', 'bpmn:IntermediateThrowEvent', 'bpmn:IntermediateCatchEvent', 'bpmn:EndEvent' ])) {
```

Comment:
Same applies for boundary events. They are catch events, too.

Reply:
Sorry about that. I fixed it